airbyte-webapp/src/pages/ConnectionPage/pages/ConnectionItemPage/components/StateBlock.tsx

```diff
@@ -28,3 +28,16 @@ export const StateBlock: React.FC<StateBlockProps> = ({ connectionId }) => {
     </Card>
   );
 };
+
+function displayState(
```

Comment:
`calculateState`? `formatState`?  This doesn't have to do with displaying it really.